components/autosize_textarea.tsx

```diff
@@ -34,39 +34,49 @@ export default class AutosizeTextarea extends React.Component<AutosizeTextareaPr
     }
 
     private getTextareaValue(key: keyof HTMLTextAreaElement) {
-        return this.textareaRef.current ? this.textareaRef.current[key] : undefined;
+        return this.textareaRef.current ? this.textareaRef.current[key] : null;
     }
 
     public get value() {
-        return this.getTextareaValue('value');
+        return this.getTextareaValue('value') as string;
```

Comment:
You should handle here the `null` case, instead of just casting to string.

Reply:
probably the best way to do this is just type checking the returned value, if is string, return it, if not return null or "" or what should be expected here.